src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/AssemblyContigAlignmentSignatureClassifier.java

```diff
@@ -92,6 +95,11 @@
     //==================================================================================================================
     // FOR ASSEMBLY CONTIGS WITH TWO ALIGNMENTS
 
+    /**
+     * Logic similar to
```

Comment:
Basically, this method a left-over from the early days developing this new tool:
initially, the classification was more fine-grained up front , even for contigs with exactly two alignments, I subdivided them so that each case could be handled respectively.

Now that things are stable, it is time to remove this fine-grained division upfront, and only leave the other two.
I have a branch for this exactly for doing this (a bunch of clean ups, including no longer creating a directory), so can I remove it there?

On the other hand, regarding the other two methods that have highly similar logics: `BreakpointsInference.getInferenceClass()`, and `SimpleNovelAdjacencyInterpreter.inferSimpleOrBNDTypesFromNovelAdjacency()`
they should be consolidated into a single function, which is now in 
`BreakpointsInference.inferFromSimpleChimera()`.